input-forms.md

```diff
@@ -0,0 +1,137 @@
+# Input-forms Endpoints
+[Back to the list of all defined endpoints](endpoints.md)
+
+## Main Endpoint
+**/api/configuration/input-forms**   
+
+Provide access to the configured input-forms. It returns the list of existent input-forms.
+
+Example: to be provided
+
+**/api/configuration/value-pairs**   
+
+Provide access to the configured value-pairs. It returns the list of existent value-pairs.
+
+Example: to be provided
+
+## Single Input-Form 
+**/api/configuration/input-forms/<:form-name>**
+
+Provide detailed information about a specific input-form. The JSON response document is as follow
+```json
+{
+  "name": "traditional",
+  "pages": [
+  {
+  	header: {"en": "First page"},
+  	mandatory: true,
+  	fields: [
+  		{
+  			label: {"en":"Authors"},
+  			repeatable: false,
+  			mandatory: null,
+  			hints: {"en":"Enter the names of the authors of this item."},
+  			input: {
+  				type: "name"
+  			}
+  			scope: null, 
+  			visibility: {
+  				main: null,
+  				other: null
+  			},
```

Comment:
It's not really clear to me what the difference is between `scope`, `visibility.main` and `visibility.other`.
* Does `scope` relate to "workspace" and "workflow"? Or the the Collection UUID?
* Or does `visibility` target "workspace" and "workflow"? But then I'm not sure what "main" and "other" mean.

Reply:
I'm trying to merge the concepts from the item-submission and the input-form adding support for some of the most common basic customization that I have seen on the submission system as well

- scope: it should be one of submission (aka workspace) or workflow. It is where the panel or the filed have it main purpose
- visibility.main can eventually be set to read-only to disallow the edit of the field / panel in the scope
- visibility.other can eventually be set to read-only to allow the field / panel to be seen also outside the scope

Reply:
I would add this to the contract document together with some example values like `hidden`, `read-only`, `admin-only`, `write`...